README.md

```diff
@@ -1,74 +1,52 @@
-# Hackathon Starter
+<img align="center" src="./docs/img/colonyStarter_combomark.svg" />
 
-This is a simple starter project that demonstrates how to use [colonyJS](https://github.com/JoinColony/colonyJS) (a JavaScript client for [colonyNetwork](https://github.com/JoinColony/colonyNetwork)). This project is set up to start a local test network using [Ganache](https://github.com/trufflesuite/ganache-cli) and then deploy the [colonyNetwork](https://github.com/JoinColony/colonyNetwork) contracts to the local test network using [Truffle](https://github.com/trufflesuite/truffle). This project also uses [TrufflePig](https://github.com/JoinColony/trufflepig) to find and read the deployed contracts during local development.
+# Colony Starter
 
-This project includes example scripts also found in the [Get Started](https://joincolony.github.io/colonyjs/docs-get-started/) and [Task Lifecycle](https://docs.colony.io/colonyjs/docs-task-lifecycle/) pages of the [colonyJS Docs](https://docs.colony.io/colonyjs/docs-overview). In order to help you hit the ground running with your new project, this starter project includes all of the examples from the docs plus some, and it gives you the opportunity to execute and test those examples with a few simple commands.
+_Learn to build with Colony! Get a head start on your next project!_ Colony Starter (formerly known as "Hackathon Starter") includes a variety of starter projects that demonstrate how to use and integrate different tools with Colony.
 
-## Prerequisites
+## Get Started
 
-- Yarn 1.7
-- Node 9.5
-
-_You may find it helpful to use Node Version Manager (`nvm`) to manage node versions._
-
-## Install
-
-Move to your working directory and clone the repository:
-
-```
-git clone https://github.com/JoinColony/hackathonStarter.git
-```
-
-Move to your new project directory:
+### Prerequisites
 
-```
-cd hackathonStarter
-```
-
-Install the project dependencies:
-
-```
-yarn
-```
+- Yarn
+- Node 9.5
 
-## Development
+_Note: You may find it helpful to use Node Version Manager (`nvm`) to manage node versions._
 
-### Start Network
+### Step 1
```

Comment:
I really like the idea of using `yarn create` but I'm pretty sure this would require renaming the package to something prefixed with `create-`. I can double check and we can consider.

As far as `npx` goes, the command that builds a project would be the same as the command in "Step 2" just with `npx` in front (ie `npx @colony/colony-starter colony-starter-basic`). I think you would still have to run through "Step 1".